minos/api_gateway/discovery/database/client.py

```diff
@@ -57,8 +50,8 @@ def __redis_connect(self):
         try:
             redis_connection = redis.Redis(host=self._redis_host, port=self._redis_port, password=self._redis_password)
             redis_connection.ping()
-        except Exception:
-            raise Exception
+        except Exception as exc:
+            raise exc
```

Comment:
As we are re-raising this exception without including any new logic, we should remove this `try-except` statement for simplicity sake